Description: Allow build to be reproducible
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-12-02
--- liberasurecode-1.1.0.orig/Makefile.am
+++ liberasurecode-1.1.0/Makefile.am
@@ -32,7 +32,8 @@ thisinclude_HEADERS = \
 	include/rs_vand/liberasurecode_rs_vand.h
 
 install-exec-hook:
-	-(cd $(DESTDIR)$(includedir) && \
+	-(mkdir -p $(DESTDIR)$(includedir) && \
+	   cd $(DESTDIR)$(includedir) && \
 	   rm -f erasurecode.h erasurecode_version.h \
 	         erasurecode_stdinc.h erasurecode_helpers.h \
 	         config_liberasurecode.h && \
